src/routes/transactions/transactions-view.controller.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Body,
+  Controller,
+  HttpCode,
+  Module,
+  Param,
+  Post,
+} from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { ConfirmationView } from '@/routes/transactions/entities/confirmation-view/confirmation-view.entity';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import {
+  TransactionDataDto,
+  TransactionDataDtoSchema,
+} from '@/routes/common/entities/transaction-data.dto.entity';
+import { TransactionsViewService } from '@/routes/transactions/transactions-view.service';
+import { DataDecodedRepositoryModule } from '@/domain/data-decoder/data-decoded.repository.interface';
+
+@ApiTags('transactions')
+@Controller({
+  path: '',
+  version: '1',
+})
+export class TransactionsViewController {
```

Comment:
We should keep in mind to generify this if we move towards more `views` endpoints.